docs/station.md

```diff
@@ -0,0 +1,49 @@
+# station (Private Beta)
+A hybrid station has the capability to trigger a scan within your Virtual Private Cloud (VPC) environment based on the inputs provided through the HCP UI.
+The `station` command is used to fetch scan jobs from [HCP](https://portal.cloud.hashicorp.com/)and execute them in your VPC. It will scan any available jobs, otherwise it will sleep and wait for a job to be queued in HCP.
+
+## Setting Up
+In order to run `station` in your environment there are 3 configuration steps required in HCP and your local environment.
+
+### Create a Service Principal
+**Note** Does the type of SP matter in this case?
+You will need to log into the HCP web portal to create your Service Principals. You can follow the HCP docs to create your [Project Service Principals](https://developer.hashicorp.com/hcp/docs/hcp/admin/iam/service-principals#project-level-service-principals-1). You will need to create the Service Principals at Admin level, more information about this level contained in [these](https://developer.hashicorp.com/hcp/docs/hcp/admin/iam/users#project-role) HCP docs.
+
+### Create a Station in the HCP Portal
+> ***Note:*** Before you create a station, there is currently now way to delete stations via the UI
+
+Now, you will need to initiate a station in the HCP web portal. If it's your first time using `Vault Radar` in HCP, you will need to configure a data source of your choice in order to get the `Station` tab view. This data source will not be used for running the station at all. This is a known UX issue that will be fixed in a future iteration.
+
+Once you have the `Station` tab view, you can start working through the station creation flow. This will give you a Station ID you will use in your local environment.
+
+### Configure your Local Environment
+To run station, the following environment variables must be set:
+
+```bash
+export HCP_PROJECT_ID=<HCP_PROJECT_ID>
+export HCP_CLIENT_ID=<HCP_CLIENT_ID>
+export HCP_CLIENT_SECRET=<HCP_CLIENT_SECRET>
+
+# The Station Id that was provided after creation
+export HCP_RADAR_STATION_ID=<STATION_ID>
+```
+
+Your HCP Project ID can be found by selecting your project --> `Project Settings` --> `Project ID`.
+
+## Running Station
+Now that our environment is configured, we're ready to run the station locally. Use the following command to start the station:
+
+```bash
+vault-radar station setup
+```
+
+This command has no local output but is still running in the background, waiting for jobs. You can follow the log file for debugging if you run into issues.
+
+## Creating a Station Resource
+Now that station is up and running, we can add a resource to be scanned. In the `Station` tab in the HCP web portal, select `Add resource to station`, which will prompt you to add a Git URL as your resource. Your locally running Station will pull this job, scan it, and upload events to HCP.
```

Comment:
we should call out that the resource should be unique to the station (ie not something already set up in SaaS)

Reply:
let's also call out that after a scan has happened the can view the resource in the Resources pane?

Reply:
Is it only after a scan that the resource will show up?

Reply:
Nvm. I just verified it'll show up after adding it.

Reply:
Do you have recommended phrasing for the first one? Is it the case that you can't have the same resource configured for both Saas and Station?

Reply:
> Do you have recommended phrasing for the first one?

`Note that resources assigned to station should be new to this project (ie not already being scanned by Radar SaaS)` (please clean that up if it's not great phrasing tho!)

>  Is it the case that you can't have the same resource configured for both Saas and Station?
I'm not sure. We'd discussed having a uniqueness check, but I think we're using an upsert API so if I had to guess we probably aren't enforcing it currently? @JoshAtHC might now more

Reply:
FYI I left a separate comment for phrasing that I like better so feel free to ignore this one. mostly leaving it open in case josh answers on "do we block it server side"